python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -751,11 +762,23 @@ async def arun(self, checkpoint=False):
                 self.total_coma_y_offset += coma_y
                 await self.hexapod_offset(focus_offset, x=coma_x, y=coma_y)
                 if self.offset_telescope:
-                    tel_x, tel_y = results["tel_offset"][0], -results["tel_offset"][1]
-                    self.log.info(f"Applying telescope offset x/y: {tel_x}/{tel_y}.")
-                    await self.atcs.offset_xy(
-                        x=tel_x,
-                        y=tel_y,
+                    tel_el_offset, tel_az_offset = (
+                        results["tel_offset"][0],
+                        -results["tel_offset"][1]
+                        * np.cos(
```

Comment:
How are we supposed to know why you're doing this?
Maybe add a comment saying that the cosine is accounting for how az offsets depend on elevation.

Reply:
cosine removed. this comment no longer applies